mdk/jira.py

```diff
@@ -35,10 +35,8 @@
 try:
     import keyring
 except:
-    # TODO Find a better way of suggesting this
-    # debug('Could not load module keyring. You might want to install it.')
-    # debug('Try `apt-get install python-keyring`, or visit http://pypi.python.org/pypi/keyring')
-    pass
+    logging.warning('Could not load module keyring. You might want to install it.')
+    logging.warning('Try `apt-get install python-keyring`, or visit http://pypi.python.org/pypi/keyring')
```

Comment:
Hi @mudrd8mz,
I'd use the http**S** version of the link, https://pypi.python.org/pypi/keyring, 😉.

HTH,
Matteo